rgw/v2/tests/s3_swift/configs/test_cloud_transition.yaml

```diff
@@ -0,0 +1,18 @@
+# script: test_cloud_transitions.py
+config:
+  bucket_count: 1
+  objects_count: 10
```

Comment:
PLease increase bucket and object count

Reply:
As discussed, changed objects_count to 100.